eventbridge-api-destinations-cloudevents-cdk-typescript/example-pattern.json

```diff
@@ -0,0 +1,56 @@
+{
+    "title": "Send CloudEvents with EventBridge API destinations",
+    "description": "Send structured or binary mode CloudEvents with EventBridge API destinations",
+    "language": "TypeScript",
+    "level": "200",
+    "framework": "CDK",
+    "introBox": {
+        "headline": "How it works",
+        "text": [
+            "This pattern demonstrates using EventBridge API destinations to convert EventBridge events to CloudEvents using binary or structured mode encoding in CloudEvents."
+        ]
+    },
+    "gitHub": {
+        "template": {
+            "repoURL": "https://github.com/aws-samples/serverless-patterns/tree/main/eventbridge-api-destinations-cloudevents-cdk-typescript",
+            "templateURL": "serverless-patterns/eventbridge-api-destinations-cloudevents-cdk-typescript",
+            "projectFolder": "eventbridge-api-destinations-cloudevents-cdk-typescript",
+            "templateFile": "lib/eventbridge-api-destinations-cloudevents-cdk-typescript-stack.ts"
```

Comment:
There is an issue when the templateFile contains the projectFolder name. Could you please rename the stack file to something that doesn't contain the folder name.